src/index.js

```diff
@@ -115,8 +115,12 @@ const main = async () => {
     post = await db.collection('posts').findOne({ _id: id })
     res.status(201).send(post)
   })
-  // retreive news and comment sorted by timestamp in reversed order
-  app.get('/posts', async (req, res) => {
+  // retreive news and comment sorted by wilson score or timestamp in reversed order
+  app.get('/posts/sorted', async (req, res) => {
```

Comment:
This should be in the same request. We will add the sort order as an argument later. Let's just sort by score by default.